optimum/exporters/tasks.py

```diff
@@ -1642,44 +1641,33 @@ def infer_library_from_model(
         if library_name is not None:
             return library_name
 
-        full_model_path = Path(model_name_or_path) / subfolder
-
-        if not full_model_path.is_dir():
-            model_info = huggingface_hub.model_info(model_name_or_path, revision=revision)
-            library_name = getattr(model_info, "library_name", None)
-
-            # sentence-transformers package name is sentence_transformers
-            if library_name is not None:
-                library_name = library_name.replace("-", "_")
```

Comment:
Why is this removed?

Reply:
Getting the library_name from the `model_info` is unreliable. I have seen that if not set in many cases this return transformers.

Reply:
I see. Ideally this could be in an other PR, but that's fine